### DISCUSSION TITLE
Стрелки или обычные функции

### DISCUSSION THREAD

**Discussion by furtivite** [doka-guide/platform#205] [Q&A]
У нас в `.eleventy.js` и дальше на платформе иногда используются анонимные функции как стрелки:

```js
config.addFilter('ruDate', (value) => { ... }
```

А иногда как функции

```js
config.addShortcode("lastModified", function (date) { ... }
```

Стоило бы договориться о правилах и привести к единому формату.

**Comment by furtivite** [comment]
Я лично за стрелки

**Comment by nlopin** [comment]
стрелки ок, если не нужен `this`

**Comment by pepelsbey** [comment]
Я бы для целей платформы нормализовал это как угодно сейчас. А потом, когда договоримся про стиль кода контента, вернул бы это и в платформу. @nlopin может перевезём твой любимый Prettier в платформу, но только для JS-файлов?

**Comment by starkovsky** [comment]
Хотел бы поддержать идею вернуть prettier хотя бы для js файлов
